lncfg/wtclient.go

```diff
@@ -1,62 +1,20 @@
 package lncfg
 
-import (
-	"fmt"
-	"strconv"
-
-	"github.com/lightningnetwork/lnd/lnwire"
-)
-
 // WtClient holds the configuration options for the daemon's watchtower client.
 type WtClient struct {
-	// PrivateTowerURIs specifies the lightning URIs of the towers the
-	// watchtower client should send new backups to.
-	PrivateTowerURIs []string `long:"private-tower-uris" description:"Specifies the URIs of private watchtowers to use in backing up revoked states. URIs must be of the form <pubkey>@<addr>. Only 1 URI is supported at this time, if none are provided the tower will not be enabled."`
```

Comment:
do we want to keep this and return an error if people still have them configured? it would be nice to get an action item as opposed to letting people debug config errors. perhaps we can deprecate fully in 0.8?

Reply:
Good point, done!